plugin/init.lua

```diff
@@ -26,12 +26,8 @@ local _smart_splits_wezterm_config = {
   },
 }
 
-local direction_keys = {
-  h = 'Left',
-  j = 'Down',
-  k = 'Up',
-  l = 'Right',
-}
+local directions = { 'Left', 'Down', 'Up', 'Right' }
+local direction_keys = {}
```

Comment:
And then leave this as the default value

```suggestion
local direction_keys = {
  h = 'Left',
  j = 'Down',
  k = 'Up',
  l = 'Right',
}
```